docs/csharp/language-reference/operators/nameof.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "nameof operator - C# reference"
+ms.custom: seodec18
+ms.date: 07/12/2019
+f1_keywords:
+  - "nameof_CSharpKeyword"
+  - "nameof"
+ms.assetid: 33601bf3-cc2c-4496-846d-f9679bccf2a7
+---
+# nameof operator (C# reference)
+
+The `nameof` operator obtains the simple name of a variable, type, or member as the string constant:
```

Comment:
We should define "simple name". Suggestion:

```suggestion
The `nameof` operator obtains the simple name of a variable, type, or member as the string constant. A [**simple name**](~/_csharplang/spec/expressions#simple-names) is the identifier name. In the case of a generic definition, it is the identifier name followed by any type parameters.
```

Reply:
@BillWagner I hoped the example would substitute the definition, which is quite difficult to craft.

For example, from the suggestion:
>In the case of a generic definition, it is the identifier name followed by any type parameters.

I would assume that type parameters are included, but the following code
```csharp
Console.WriteLine(nameof(List<int>));
```

produces 
```console
List
```

I've looked at the spec now. It doesn't actually say "simple name", but just a "name". So i suggest to remove the word "simple":
>The `nameof` operator obtains the name of a variable, type, or member as the string constant:

Reply:
@BillWagner I think I was wrong with the word choice in the first version of the article. Could you please take a look at the updated version.

Additionally, I've added the language version, in which the operator was added, and the link from the "What's new..." section.